components/Activity/ActivityBoxStyles.ts

```diff
@@ -0,0 +1,63 @@
+import { Dimensions } from "react-native"
+import EStyleSheet from "react-native-extended-stylesheet"
+
+const { height, width } = Dimensions.get("window")
+const smallScreen = width < 563
+const ActivityBoxStyles = EStyleSheet.create({
+  ActivityBoxContainer: {
+    width: smallScreen ? Dimensions.get("window").width : "100%",
```

Comment:
Small suggestion. Just use `width` and `height` and since you already destructured both values on line 4.

Reply:
Done. Thanks